nntrainer/src/layers.cpp

```diff
@@ -141,17 +151,30 @@ Tensor InputLayer::forwarding(Tensor in) {
   return input;
 }
 
-void InputLayer::initialize(int b, int h, int w, int id, bool init_zero,
-                            WeightIniType wini) {
+int InputLayer::initialize(int b, int h, int w, int id, bool init_zero,
+                           WeightIniType wini) {
+  int status = ML_ERROR_NONE;
+  if (b <= 0 || h <= 0 || w <= 0) {
```

Comment:
I think it's better for those variables to be unsigned if they are always positive (or zero) values.

Reply:
Yes. I will change them later commit. Thanks.